PetAdoptionNG/src/app/components/create-application/create-application.component.html

```diff
@@ -1,14 +1,13 @@
 <p>create-application works!</p>
 
-
 <div>
     <div>
         <span>Children </span>
         <input [(ngModel)] = "children" type="number">
     </div>
     <div>   
         <span>Pet ID </span>
-    <input [(ngModel)] = "petID" type="number">
+    <input [(ngModel)] = "petID" type="number" placeholder={{petID}}> <h3>{{pet}}</h3>
     </div>
```

Comment:
Why do you have to have them input the pet's ID? I thought that was generated automatically by SQL. Or is there something else going on that I don't see?